hera_qm/utils.py

```diff
@@ -31,6 +31,9 @@ def get_metrics_ArgumentParser(method_name):
                        help='Modified z-score cut for most cross-polarized antenna. Default 5 "sigmas"')
         a.add_argument('--deadCut', default=5.0, type=float,
                        help='Modified z-score cut for most likely dead antenna. Default 5 "sigmas"')
+        a.add_argument('--alwaysDeadCut', default=1.0, type=float,
```

Comment:
Should we have `default=10.` instead? That's what the help suggests.

Reply:
Oops. My bad. Yes.